steampy/client.py

```diff
@@ -311,10 +316,77 @@ def fetch_price(self, item_hash_name: str, game: GameOptions, currency: str = Cu
+        if response.status_code == 200:
+            assets_descriptions = json.loads(text_between(response.text, "var g_rgAssets = ", ";"))
+            listing_id_to_assets_address = get_listing_id_to_assets_address_from_html(response.text)
+            listings = get_market_listings_from_html(response.text)
+            listings = merge_items_with_descriptions_from_listing(listings, listing_id_to_assets_address,
+                                                                  assets_descriptions)
+            return listings
+        else:
+            raise FetchException("Steam market page return a %s" % response.status_code)
+
+    @login_required
+    def create_sell_listing(self, assetid: str, game: GameOptions, money_to_receive: int) -> dict:
+        data = {
+            "assetid": assetid,
+            "sessionid": self._get_session_id(),
+            "contextid": game.context_id,
+            "appid": game.app_id,
+            "amount": 1,
+            "price": money_to_receive
+        }
+        headers = {'Referer': "http://steamcommunity.com/profiles/%s/inventory" % self.steam_guard['steamid']}
+        response = self._session.post("https://steamcommunity.com/market/sellitem/", data, headers=headers).json()
+        if response.get("needs_mobile_confirmation"):
+            return self._confirm_sell_listing(assetid)
+        return response
+
+    @login_required
+    def create_buy_order(self, market_name: str, price_single_item: int, quantity: int, game: GameOptions,
+                         currency: Currency = Currency.USD) -> dict:
+        data = {
+            "sessionid": self._get_session_id(),
+            "currency": currency.value,
+            "appid": game.app_id,
+            "market_hash_name": market_name,
+            "price_total": price_single_item * quantity,
+            "quantity": quantity
+        }
+        headers = {'Referer': "http://steamcommunity.com/market/listings/%s/%s" % (game.app_id, market_name)}
+        response = self._session.post("https://steamcommunity.com/market/createbuyorder/", data, headers=headers)
+        response = response.json()
+        return response
+
+    @login_required
+    def remove_sell_listing(self, sell_listing_id: str) -> dict:
+        data = {"sessionid": self._get_session_id()}
+        headers = {'Referer': "http://steamcommunity.com/market/"}
+        url = "http://steamcommunity.com/market/removelisting/%s" % sell_listing_id
+        response = self._session.post(url, data=data, headers=headers).json()
+        return response
+
+    @login_required
+    def cancel_buy_order(self, buy_order_id) -> dict:
+        data = {
+            "sessionid": self._get_session_id(),
+            "buy_orderid": buy_order_id
+        }
+        headers = {"Referer": "http://steamcommunity.com/market"}
+        response = self._session.post("http://steamcommunity.com/market/cancelbuyorder/", data, headers=headers).json()
+        return response
+
```

Comment:
The request in "cancel_buy_order" it return nothing even if the request works. So response will always be an empty dict (except when there are errors), I am not sure what to return in this case, maybe just the response object?

PS. these last two methods are called "remove" and "cancel" because it's how Steam call them, I could change them.